### PR TITLE
Cleanup shell scripts

### PR DESCRIPTION
**Pull request by jperryhouts**
Some things I came across while browsing the source code.

### CODE REVIEW

File: tests/default.sh [hunk 1]
```diff
@@ -1,2 +0,0 @@
-#!/bin/sh
-cat
```
Comment (bangerth) [inline tests/default.sh]: where is/was this used?

File: benchmark/inclusion/run.sh [hunk 2]
```diff
@@ -10,11 +10,11 @@ cp global.prm.base temp.prm
 echo "subsection Mesh refinement" >>temp.prm
 echo "set Initial global refinement = $r" >> temp.prm
 echo "end" >> temp.prm
-../../aspect temp.prm | grep Error
```
Comment (tjhei) [inline benchmark/inclusion/run.sh]: what is the reason for changing the path?

File: tests/default.sh (hunk 1 above)
Comment (tjhei) [inline tests/default.sh]: This file is required (default filtering script)

Comment (tjhei) [inline tests/default.sh]: I should probably add a comment in it that nobody is trying to delete it :-)

File: benchmark/inclusion/run.sh (hunk 2 above)
Comment (jperryhouts) [inline benchmark/inclusion/run.sh]: It seems more reasonable to assume that the user has aspect in their environment PATH than in this particular spot.

**Comment by jperryhouts** [comment]
I guess I didn't check these carefully enough before submitting the PR. Should I close this?

File: benchmark/inclusion/run.sh (hunk 2 above)
Comment (tjhei) [inline benchmark/inclusion/run.sh]: Hrm, that is something worth thinking about.

**Pull request closed without merging**